app/utilities/filterCompanyEmployees.js

```diff
@@ -0,0 +1,16 @@
+export function filterCompanyEmployees(fetchedCompanies, fetchedUsers) {
+  function filterUsersByCompany(users, companyUri) {
+    return users.filter(function (user) {
+      return user.uris.company === companyUri;
+    });
+  }
+
+  const combinedArrays = fetchedCompanies.map(function (company) {
+    return {
+      ...company,
+      employees: filterUsersByCompany(fetchedUsers, company.uri),
+    };
+  });
+
+  return combinedArrays;
```

Comment:
The `combinedArrays` variable is unnecessary. WebStorm surely tells you that.
![image](https://github.com/user-attachments/assets/e0de5446-451d-444d-87c0-728a81d49c85)